rasa_nlu/training_data/util.py

```diff
@@ -24,3 +26,22 @@ def check_duplicate_synonym(entity_synonyms, text, syn, context_str=""):
     if text in entity_synonyms and entity_synonyms[text] != syn:
         logger.warning("Found inconsistent entity synonyms while {0}, overwriting {1}->{2}"
                        "with {1}->{2} during merge".format(context_str, text, entity_synonyms[text], syn))
+
+
+def generate_lookup_regex(file_path):
+    """creates a regex out of the contents of a lookup table file"""
+    lookup_elements = []
+    with io.open(file_path, 'r') as f:
+        for line in f:
+            new_element = line.strip()
+            if new_element != '':
+                lookup_elements.append(new_element)
+    regex_string = '(?i)(\\b' + '\\b|\\b'.join(lookup_elements) + '\\b)'
```

Comment:
please add a comment that this is a regex matching for word boundaries

Reply:
added